kvirt/providers/ibm/__init__.py

```diff
@@ -55,30 +57,30 @@ def __init__(self, iam_api_key, access_key_id, secret_access_key, region, zone,
             aws_secret_access_key=secret_access_key
         )
         self.global_tagging_service = GlobalTaggingV1(authenticator=self.authenticator)
-        self.global_tagging_service.set_service_url('https://tags.global-search-tagging.cloud.ibm.com')
+        self.global_tagging_service.set_service_url(
```

Comment:
any reason not to fit this on a single line, the convention in this repo are lines of up to 120 chars